lib/ansible/modules/cloud/amazon/route53_health_check.py

```diff
@@ -156,7 +156,7 @@ def find_health_check(conn, wanted):
     """Searches for health checks that have the exact same set of immutable values"""
     for check in conn.get_list_health_checks().HealthChecks:
         config = check.HealthCheckConfig
-        if config.get('IPAddress') == wanted.ip_addr and config.get('FullyQualifiedDomainName') == wanted.fqdn and config.get('Type') == wanted.hc_type and config.get('RequestInterval') == str(wanted.request_interval):
+        if config.get('IPAddress') == wanted.ip_addr and config.get('FullyQualifiedDomainName') == wanted.fqdn and config.get('Type') == wanted.hc_type and config.get('RequestInterval') == str(wanted.request_interval) and config.get('Port') == str(wanted.port):
```

Comment:
The logic is fine but this line is ridiculously long (it was before, now it's even longer)

Can you put the if condition in brackets and add a line break or two (the brackets mean you don't need an escaping `\` at the end of the line.)